woocommerce/woocommerce-gutenberg-products-block
# Mini Cart Contents: Refactor inner blocks styling using `currentColor`

This PR refactors the style of Mini Cart Contents inner blocks to make them work better with color settings of Mini Cart Contents.

Note: This doesn't include style for quantity input which is shared between Cart and Mini Cart.

### Testing

#### Automated Tests
### Manual Testing

How to test the changes in this Pull Request:

1. Edit the Mini Cart template part.
2. Set color and background color to custom values.
3. See the inner blocks applied correctly on both editor and front end.
### User Facing Testing
These are steps for user testing (where "user" is someone interacting with this change that is not editing any code).
